src/Components/Login/Login.tsx

```diff
@@ -144,12 +147,12 @@ const Login = ( { handleSetUser }: { handleSetUser: (user: any) => void}) => {
             value={userLocation}
             onChange={event => setUserLocation(event.target.value)}
             />}
-          {activeAccount === true && <input type="submit" value="Submit" className='login-btn' />}
-          {activeAccount === false && <input type="submit" value="Submit" className='create-account-btn' />}
+          {activeAccount === true && <input type="submit" value="Log In" className='login-btn' />}
```

Comment:
This was a super sleek way of having the user know what form they are on. Awesome job!!